src/api/custom-resolvers/mutations/account-sign-in/account-checks/index.ts

```diff
@@ -39,9 +39,7 @@ const accountChecks = async (context: Context, account: Account, urlOrigin: stri
     if (!account.status.isVerified) {
       console.info('Unable to sign in account - account has not been verified yet');
 
-      const now = new Date();
-
-      const verificationHasExpired = isAfter(now, account.verificationExpiry);
+      const verificationHasExpired = dateIsInThePast(account.verificationExpiry);
```

Comment:
Nice helper, could we update the other 3x instances `isAfter(now ...` to use this?

If you search for:
```ts
= isAfter(now,
```